scripts/appleseedMaya/renderGlobals.py

```diff
@@ -405,20 +406,24 @@ def create(self):
                         adaptiveSampling = mc.getAttr("appleseedRenderGlobals.adaptiveSampling")
 
                         self._addControl(
-                            ui=pm.intFieldGrp(
-                                label="Min Samples", numberOfFields=1, enable=adaptiveSampling),
+                            ui=pm.intSliderGrp(
+                                label="Min Samples", field=True, value=16, cw=(3,160), minValue=0, 
+                                fieldMinValue=0, maxValue=256, fieldMaxValue=1000000, enable=adaptiveSampling),
                             attrName="minPixelSamples")
                         self._addControl(
-                            ui=pm.intFieldGrp(
-                                label="Max Samples", numberOfFields=1),
+                            ui=pm.intSliderGrp(
+                                label="Max Samples", field=True, value=256, cw=(3,160), minValue=16, 
+                                fieldMinValue=0, maxValue=1024, fieldMaxValue=1000000),
                             attrName="samples")
                         self._addControl(
-                            ui=pm.intFieldGrp(
-                                label="Batch Sample Size", numberOfFields=1, enable=adaptiveSampling),
+                            ui=pm.intSliderGrp(
+                                label="Batch Sample Size", field=True, value=8, cw=(3,160), minValue=1, 
+                                fieldMinValue=1, maxValue=128, fieldMaxValue=1000000, enable=adaptiveSampling),
                             attrName="batchSampleSize")
                         self._addControl(
-                            ui=pm.floatFieldGrp(
-                                label="Noise Threshold", numberOfFields=1, enable=adaptiveSampling),
+                            ui=pm.floatSliderGrp(
+                                label="Noise Threshold", field=True, value=0.1, step=0.01, precision=3, cw=(3,160), minValue=0.0001, 
```

Comment:
Step should be 0.02, I think precision should be 4 (four digits after the decimal point), hard max value should be 10000.

Reply:
ok